### PR TITLE
fixed golint error in pkg/apis/admissionregistration/validation

### PR DESCRIPTION
**Pull request by joremysh**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fixed golint eror in pkg/apis/admissionregistration

**Which issue(s) this PR fixes**:

Part of #68026

**Special notes for your reviewer**:
run hack/update-bazel.sh and hack/update-gofmt.sh ok

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by neolit123** [comment]
> Fixes #68026
please change this to something like:

> Part of #68026

or it will close the referenced ticket.

**Review by neolit123 — COMMENTED** [review]
/ok-to-test
/priority backlog
/sig auth

File: pkg/apis/admissionregistration/v1/defaults.go [hunk 1]
```diff
@@ -27,7 +27,8 @@ func addDefaultingFuncs(scheme *runtime.Scheme) error {
 	return RegisterDefaults(scheme)
 }
 
-func SetDefaults_ValidatingWebhook(obj *admissionregistrationv1.ValidatingWebhook) {
```
Comment (liggitt) [inline pkg/apis/admissionregistration/v1/defaults.go:30]: The name of defaulting methods is fixed and cannot be changed without also changing the generated defaulters. That is not currently planned, due to the wide ripples in all codebases that depend on generated defaults.

File: hack/.golint_failures [hunk 2]
```diff
@@ -6,9 +6,6 @@ cmd/kubeadm/app/apis/kubeadm/v1beta2
 pkg/apis/abac/v0
 pkg/apis/abac/v1beta1
 pkg/apis/admission
-pkg/apis/admissionregistration/v1
-pkg/apis/admissionregistration/v1beta1
-pkg/apis/admissionregistration/validation
```
Comment (liggitt) [inline hack/.golint_failures:11]: the validation package can be resolved, but the defaulting functions in the v1/v1beta1 packages should be left as-is

**Review by joremysh — COMMENTED** [review]
Only update in comments now.

File: hack/.golint_failures (hunk 2 above)
Comment (JeremyShih) [reply hack/.golint_failures:11]: updated.

File: pkg/apis/admissionregistration/v1/defaults.go (hunk 1 above)
Comment (JeremyShih) [reply pkg/apis/admissionregistration/v1/defaults.go:30]: The changes on names of method are all removed now.

**Comment by JeremyShih** [comment]
Hi @liggitt could you give the PR a milestone?

**Pull request merged**